### ISSUE TITLE
"Unsupported version of Among Us: 2020.12.5", everything updated

### ISSUE THREAD

**Issue by VinceCaso** [ottomated/CrewLink#282]
Hey folks! I've recently uninstalled / reinstalled the latest release of both Among Us and CrewLink, and regardless still get the following error message when opening CrewLink:

"You are on an unsupported version of Among Us: 2020.12.5."

Does anyone know a fix for this?

**Comment by Ornamus** [comment]
Yup! That's actually caused by failing to connect to the voice server - the default one is down at the moment. If you host your own, or connect to another one (there's a couple available in #279) you should be able to play as normal.

**Comment by VinceCaso** [comment]
Ahh that makes sense. I deployed my own via Heroku though, would that still not work as well?

**Comment by Ornamus** [comment]
Oh interesting! If you've deployed it since the latest server update, then it should work. If not, something else is going wrong 🤔

**Comment by VinceCaso** [comment]
I deployed the Herokuapp last week, I believe, so it might have been before the server update. Would it need to be re-deployed each time?

**Comment by Ornamus** [comment]
I think the server update was made a couple days ago, so that's probably it.

I've found the fastest way is to just delete my server then deploy a new one with the same name.

**Comment by TheShadowEevee** [comment]
Did you ever solve the issue? There's been a few updates since then you could try again with if not.
